app/serializers/school_certifying_official_serializer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class SchoolCertifyingOfficialSerializer < ActiveModel::Serializer
+  attribute :facility_code
+  attribute :institution_name
```

Comment:
We don't need the above two attributes, right?  `facility_code` and `institution_name` are returned  via the `institutions` object/serializer.

Reply:
@jholton You are correct, I just pushed a commit removing those two attributes from the SchoolCertifyingOfficialSerializer